### PR TITLE
Quick add

### PR DESCRIPTION
**Pull request by PsychicMuffin**
The CSS still needs some work, and the enter button should probably add the item instead of just selecting the item, preferably keeping the focus on the quick add so you can go through a lot of items at once.

### CODE REVIEW

File: js/app.js [hunk 1]
```diff
@@ -1,11 +1,32 @@
-angular.module('PathOfDamage', [])
-.controller('Damage', function ($scope, $document, $window, DataService) {
+angular.module('PathOfDamage', ['ngSanitize', 'ui.select'])
+.controller('Damage', function ($scope, $document, $window, DataService, Items) {
+  const ELEMENTS = ['physical', 'fire', 'cold', 'lightning', 'chaos'];
```
Comment (PsychicMuffin) [inline js/app.js]: I don't want to duplicate this const, but I think the correct way is to pass it into the DataService during its construction probably, and that tripped me up when I was messing with it.

File: js/Items.js [hunk 2]
```diff
@@ -0,0 +1,99 @@
+angular.module('PathOfDamage')
```
Comment (PsychicMuffin) [inline js/Items.js:1]: I feel like I go through a lot of levels of indentation just to return a single array, but I didn't find a cleaner way to do this.

File: js/Items.js [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+        {id: 69, name: 'Arctic Armour (Fire)', value: -12, section: 'taken', table: 'more', elements: [1]},
+
+        // Damage Taken From Mana Before Life
+        {id: 70, name: 'Mind Over Matter', value: 30, section: 'shift', table: 'mana', elements: [0,1,2,3,4]},
+        {id: 71, name: 'Divine Guidance', value: 10, section: 'shift', table: 'mana', elements: [0,1,2,3,4]},
+        {id: 72, name: 'Cloak of Defiance', value: 20, section: 'shift', table: 'mana', elements: [0,1,2,3,4]},
+        {id: 73, name: 'The Aylardex', value: 6, section: 'shift', table: 'mana', elements: [0,1,2,3,4]},
+        {id: 74, name: 'Mind of the Council', value: 30, section: 'shift', table: 'mana', elements: [3]},
+        {id: 75, name: 'Damage taken from Mana Corruption', value: 6, section: 'shift', table: 'mana', elements: [0,1,2,3,4]}
+      ].sort(function (a, b) {
```
Comment (PsychicMuffin) [inline js/Items.js:94]: This should probably be a map eventually (faster lookups when I implement encoding modifiers as IDs), and we then return map.values.sort or something.

File: js/Items.js (hunk 2 above)
Comment (ficocelliguy) [reply js/Items.js:1]: I don't think its an issue. A getter is good styling and the rest of the indents are angular artifacts.

**Review by ficocelliguy — APPROVED** [review]

**Pull request merged**